monitoring/snippets/v3/uptime-check-client/snippets.py

```diff
@@ -161,12 +216,11 @@ def project_id():
     return project_id
 
 
-def project_name():
+def project_name() -> str:
     return "projects/" + project_id()
```

Comment:
Could stand a docstring, even though it's so short and simple.

Reply:
I would say it is self descriptive. Will see if it will be simpler to inline this code